sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multilabel-in-pipeline/data/validation-mltable-folder/validation_annotations.jsonl

```diff
@@ -0,0 +1,25 @@
+{"image_url": "azureml://subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourcegroups/ayush_mishra_res01/workspaces/mlw-basic-prod-hrishi/datastores/workspaceblobstore/paths/LocalUpload/75da6dddd4a47a30707c8c4fc737f5b9/multilabelFridgeObjects/images/5.jpg", "label": ["water_bottle"]}
```

Comment:
is this needed?

Reply:
Nope, it was created while checking the code . Thanks for pointing it out.
Removed